internal/formats/common/spdxhelpers/to_syft_model.go

```diff
@@ -42,6 +49,31 @@ func ToSyftModel(doc *spdx.Document2_2) (*sbom.SBOM, error) {
 	return s, nil
 }
 
+// NOTE(jonas): this is a hack, SPDX doesn't seem to support a way
```

Comment:
Might not call this "a hack", just "this is our best effort to determine the scheme" or something